worlds/gl/Rom.py

```diff
@@ -27,6 +27,7 @@ def get_base_rom_as_bytes() -> bytes:
     Raises an exception if the ROM doesn't match the expected hash.
     """
     try:
+        get_settings().gl_options.rom_file.validate()
         with open(get_settings().gl_options.rom_file, "rb") as infile:
```

Comment:
should use the World classattr when possible
```suggestion
        from . import GauntletLegendsWorld
        GauntletLegendsWorld.settings.rom_file.validate()
        with open(GauntletLegendsWorld.settings.rom_file, "rb") as infile:
```